Makefile

```diff
@@ -448,6 +449,11 @@ openstack: operator_namespace openstack_prep ## installs the operator, also runs
 	$(eval $(call vars,$@,openstack))
 	oc apply -f ${OPERATOR_DIR}
 
+.PHONY: openstack_wait
```

Comment:
We also have `OPERATOR_NAME=openstack [make wait](https://github.com/openstack-k8s-operators/install_yamls/blob/main/Makefile#L342)` target to check the operator installation status. Do we want to re-use that here?